include/pelz_request_handler.h

```diff
@@ -62,6 +62,7 @@ extern "C"
 {
 #endif
 
+#ifndef PELZ_SGX_UNTRUSTED
 /**
  * <pre>
  * Wrapper function that handles making the right function call to pass
```

Comment:
This function isn't a wrapper anymore. It actually implements the request handling.

Reply:
The function comment has been updated.